python-avd/pyavd/schema/generate_docs/utils.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ..metaschema.meta_schema_model import AristaAvdSchema, AvdSchemaField
+
+
+def list_tables(schema: AristaAvdSchema) -> set[str]:
+    """
+    Returns list of tables to render.
+
+    - Checks every root key for it's own table, and if not defined it adds a default table name using the key.
+    - Scans every root key for decendant tables.
```

Comment:
couple of typos
```suggestion
    - Checks every root key for its own table, and if not defined adds a default table name using the key.
    - Scans every root key for descendant tables.
```

When reading the two bullets I am not sure what this function does and also it looks like we could do withtout `list_tables` looking at the inner code

Reply:
This was correct before I simplified everything. simplifying the docstring too :)